src/utils/interaction.js

```diff
@@ -1,25 +1,42 @@
-import { items, displayTask } from './structure.js';
+/* eslint-disable import/no-cycle */
+import { items, render } from './structure.js';
 
-const listInput = displayTask();
+function textDecoration(listInput) {
+  listInput.forEach((item) => {
+    if (item.hasAttribute('checked')) {
+      item.nextSibling.style.textDecoration = 'line-through';
+    } else {
+      item.nextSibling.style.textDecoration = 'none';
+    }
+  });
+}
 
-function userInteraction() {
+function userInteraction(listInput) {
```

Comment:
- [ ] Please change this and all other regular functions into arrow functions, following the Javascript best practices guide.